locale/en/blog/release/v14.21.1.md

```diff
@@ -0,0 +1,43 @@
+---
+date: 2022-11-04T17:51:52.399Z
+version: 14.21.1
+category: release
+title: Node v14.21.1 (LTS)
+slug: node-v14-21-1
+layout: blog-post.hbs
+author: Bethany Nicolle Griggs
+---
+
+### Notable changes
+
+The following CVEs are fixed in this release:
+
+* **[CVE-2022-43548](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-43548)**: DNS rebinding in --inspect via invalid octal IP address (Medium)
+
+More detailed information on each of the vulnerabilities can be found in [November 2022 Security Releases](https://nodejs.org/en/blog/vulnerability/november-2022-security-releases/) blog post.
+
+### Commits
+
+* \[[`2b433af094`](https://github.com/nodejs/node/commit/2b433af094)] - **inspector**: harden IP address validation again (Tobias Nießen) [nodejs-private/node-private#354](https://github.com/nodejs-private/node-private/pull/354)
+
+Windows 32-bit Installer: https://nodejs.org/dist/v14.21.1/node-v14.21.1-x86.msi<br>
+Windows 64-bit Installer: https://nodejs.org/dist/v14.21.1/node-v14.21.1-x64.msi<br>
+Windows 32-bit Binary: https://nodejs.org/dist/v14.21.1/win-x86/node.exe<br>
+Windows 64-bit Binary: https://nodejs.org/dist/v14.21.1/win-x64/node.exe<br>
+macOS 64-bit Installer: https://nodejs.org/dist/v14.21.1/node-v14.21.1.pkg<br>
+macOS Intel 64-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-darwin-x64.tar.gz<br>
+Linux 64-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-linux-x64.tar.xz<br>
+Linux PPC LE 64-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-linux-ppc64le.tar.xz<br>
+Linux s390x 64-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-linux-s390x.tar.xz<br>
+AIX 64-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-aix-ppc64.tar.gz<br>
+ARMv7 32-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-linux-armv7l.tar.xz<br>
+ARMv8 64-bit Binary: https://nodejs.org/dist/v14.21.1/node-v14.21.1-linux-arm64.tar.xz<br>
+Source Code: https://nodejs.org/dist/v14.21.1/node-v14.21.1.tar.gz<br>
+Other release files: https://nodejs.org/dist/v14.21.1/<br>
+Documentation: https://nodejs.org/docs/v14.21.1/api/
+
+### SHASUMS
+
+```
+[INSERT SHASUMS HERE]
```

Comment:
Shasums are missing.

Reply:
Guess Beth won't be back until 2 hours from now. Can we merge it and then she updates that when she gets the chance?

Reply:
@RafaelGSS I've pushed a fixup commit that adds the shasums.